tools/wptserve/wptserve/response.py

```diff
@@ -776,10 +776,13 @@ def write_content_file(self, data):
             if not buf:
                 break
```

Comment:
First, initialize success to True before `while`.

```suggestion
                success = False
                break
```